lib/ansible/module_utils/facts/system/service_mgr.py

```diff
@@ -144,6 +144,8 @@ def collect(self, module=None, collected_facts=None):
                 service_mgr_name = 'systemd'
             elif os.path.exists('/etc/init.d/'):
                 service_mgr_name = 'sysvinit'
+            elif os.path.exists('/etc/dinit.d/'):
```

Comment:
Please create a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment). See [this fragment](https://github.com/ansible/ansible/blob/c2f528b7686c9573aa8493357219e0bbe4416dd3/changelogs/fragments/reboot-change-default-boot-command.yaml) as an example.

Also, a test for this change.

Reply:
I'd appreciate if someone else could do that. I'd have opened an issue instead of committing the change myself, but the issue form was more convoluted than the PR form, which is why I ended up here at all.

This is a trivial change after all.

Reply:
This isn't something we can reasonably test with our integration tests, so just a changelog fragment will suffice.